docs/sources/flow/reference/cli/run.md

```diff
@@ -90,11 +91,12 @@ must be configured to accept connections on `--server.http.listen-addr` and the
 address defined or inferred in `--cluster.advertise-address`.
 
 If the `--cluster.advertise-address` flag is not explicitly set, the agent
-tries to infer a suitable one from the `eth0` and `en0` local network
-interfaces. If the advertised address cannot be determined, the agent will
-fail to start. Windows users must explicitly pass a value for
-`--cluster.advertise-address`, since Windows does not have interfaces name
-`eth0` or `en0`.
+tries to infer a suitable one from `--cluster.advertise-interfaces`.
+If `--cluster.advertise-interfaces` is not explicitly set, the agent will
+infer one from the `eth0` and `en0` local network interfaces.
+If the advertised address cannot be determined, the agent will fail to start.
```

Comment:
```suggestion
The agent will fail to start if it can't determine the advertised address.
```

We could rework this sentence to make it active instead of passive.